.github/workflows/skrpro.yml

```diff
@@ -50,11 +50,11 @@ jobs:
     steps:
 
     # First, cancel unfinished jobs.
-    - uses: technote-space/auto-cancel-redundant-job@v1
+    - uses: technote-space/auto-cancel-redundant-job@v1.7.5
 
     # Based on .github/workflows/test-builds.yml
-    - name: Select Python 3.7
-      uses: actions/setup-python@v1
+    - name: Select Python 3.9
+      uses: actions/setup-python@v4
       with:
         python-version: '3.7' # Version range or exact version of a Python version to use, using semvers version range syntax.
```

Comment:
you didn't change the actual python-version to 3.9, only the title.

Reply:
Interesting! Thank you, let me give it a shot.